File: CommonLib/Objects/Event.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CommonLib.Objects
{
    public class Event
    {
        public int id { get; set; }
        public int gameId { get; set; }
        public string eventTitle { get; set; }
        public string status { get; set; }
        public string team1 { get; set; }
        public string team2 { get; set; }

        public string matchTime { get; set; }

        public IList<Outcomes> outcomesWinner { get; set; }
        public IList<Scores> scores { get; set; } 

    }

    public class Outcomes
    {
        public string name { get; set; }
        public decimal? win1 { get; set; }
        public decimal? win2 { get; set; }
        public decimal? X { get; set; }
    }

    public class Scores
    {
        public string name { get; set; }
        public int team1 { get; set; }
        public int team2 { get; set; }
    }


}

File: bkService/ClientApp/src/app/Entity/event.ts
export class Event {
  id: number;
  gameId: number;
  eventTitle: string;
  status: string;
  matchTime: string; 
  team1: string;
  team2: string;
  score: string;

  outcomes: Outcomes;

  get SportName(): string {

    var result: string;

    if (this.gameId == 33) {
      result = "Football";
    }

    if (this.gameId == 31) {
      result = "Hockey";
    }


    return result;
  }
}

export class Outcomes {
  team1: number;
  team2: number;
  x: number;
}
